modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -266,6 +266,11 @@ I::IStream& operator>> (I::IStream& is,       cv::GOpaqueDesc &) {return is;}
 I::OStream& operator<< (I::OStream& os, const cv::GArrayDesc &) {return os;}
 I::IStream& operator>> (I::IStream& is,       cv::GArrayDesc &) {return is;}
 
+#if !defined(GAPI_STANDALONE)
+I::OStream& operator<< (I::OStream& os, const cv::UMat &) { return os; }
+I::IStream& operator >> (I::IStream& is, cv::UMat &) { return is; }
```

Comment:
1) Can you please fix alignment here?
2) These operations silently do nothing, I think we could throw exceptions here instead

Reply:
Done